PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PaymentButton.kt

```diff
@@ -72,6 +72,27 @@ abstract class PaymentButton<C : PaymentButtonColor> @JvmOverloads constructor(
             prefixTextView.visibility = prefixTextVisibility
         }
 
+    /**
+     * Custom corner radius
```

Comment:
Can we add a note to what the default is if not set?

Reply:
Sorry to clarify - what will the corner radius of the button be if the this is not set?

Reply:
No worries, this would be set to null.